libr/bin/format/elf/elf.c

```diff
@@ -3967,20 +3966,11 @@ char *Elf_(r_bin_elf_compiler)(ELFOBJ *bin) {
 		free (buf);
 		return NULL;
 	}
-
+	const size_t buflen = strlen (buf);
+	char *nullbyte = buf + buflen;
+	if (nullbyte[1] && buflen < sz) {
+		nullbyte[0] = ' ';
```

Comment:
Shall we keep using `/`? There would be less tests to fix and it will be more consistent with current behaviour.

Reply:
Also, why are we doing this? Didn't you say in https://github.com/radareorg/radare2/issues/15858#issuecomment-576242135 that we should stop at the first null byte?